### PR TITLE
add examples of InferenceData common manipulations 

### PR DESCRIPTION
**Pull request by aloctavodia**
This adds a new notebook with a few examples. Is by no means exhaustive but I think is better to have something than nothing and then it will be easier to keep adding more examples.

This also renames the already existing notebook InferenceDataCookbook to InferenceDataIO and use the name InferenceDataCookbook for the new one.

EDIT: fixes partially issue #1486

### CODE REVIEW

**Review by OriolAbril — CHANGES_REQUESTED** [review]
We don't have reviewnb here so I'll do my best to clarify where is each comment referring to.

First of all, the notebook titles have to be updated. The toctree is created using the file names, but sphinx then shows the file _title_ when generating the html docs. Currently both files have InferenceData cookbook as title. I personally would suggest changing both titles. We have had the cookbook on the website for quite a long time and most users are probably familiar with it, I think leaving the same title but with different content will be confusing, changing everything will send the "nice updated docs!" message. How about "creating InferenceData" and "working with InferenceData"?

When calculating the mean in the last example, the first cell has `idata.posterior` instead of `idata.posterior.mean()`.

You'll see that in the cookbook there is a strange `(cookbook)=` thing. This is equivalent to the `.. cookbook:` rst anchors. I would add one to the new notebook to easily link there from other parts of the documentation like the InferenceData api docs

File: doc/source/getting_started/index.md [hunk 1]
```diff
@@ -5,5 +5,13 @@
...
+CreatingInferenceData
+WorkingWithInferenceData
 ```
+
+     4D            2C       2C         4D
+A 1001101 años del 101100 y 101100 del 1001101.
+
+
+44 2C
+77 4D
```
Comment (OriolAbril) [inline doc/source/getting_started/index.md:17]: looks like something got copy pasted unintentionally

**Review by OriolAbril — COMMENTED** [review]
I think all `from_xyz` link to `cookbook` anchor on their api docs, i.e. https://github.com/arviz-devs/arviz/blob/idata_examples/arviz/data/io_pymc3.py#L516

File: doc/source/getting_started/index.md (hunk 1 above)
Comment (aloctavodia) [reply doc/source/getting_started/index.md]: haha, at least is not something sensitive like my bank account password, haha

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**